common/src/main/kotlin/demo/gos/common/Players.kt

```diff
@@ -0,0 +1,32 @@
+package demo.gos.common
+
+import java.security.SecureRandom
+import kotlin.math.min
+
+object Players {
+    private val RND = SecureRandom()
+
+    fun walk(pos: Point, dest: Point, speed: Double, accuracy: Double, delta: Double): Point {
+        // Speed and angle are modified by accuracy
+        val segToDest = Segment(pos, dest)
+        // maxSpeed avoids stepping too high when close to destination
+        val maxSpeed = min(segToDest.size() / delta, speed)
+        // minSpeed must be kept <= maxSpeed
+        val minSpeed = min(maxSpeed, accuracy * speed)
+        val speed = delta * (minSpeed + RND.nextDouble() * (maxSpeed - minSpeed))
+        val relativeMove = randomishSegmentNormalized(accuracy, segToDest).mult(speed)
+        return pos.add(relativeMove)
+    }
+
+    fun spawnIn(area: Area): Point {
```

Comment:
I think that one can go directly in `Area`, it's going to be useful not only for players but for anything spawnable (weapons...)